### ISSUE TITLE
Suggestion: packed-overlay gallery

### ISSUE THREAD

**Issue by Extarys** [SemanticMediaWiki/SemanticResultFormats#369]
It seems impossible to make a `<gallery mode="packed-overlay">` but I think they are beautiful.

Would it be possible to add an option to make these?

**Comment by kghbln** [comment]
Thanks for filing an enhancement to the gallery format. Yeah, I am not aware of this result format being able to do this. Looks nice. At the same time [other enhancements](https://en.wikipedia.org/wiki/Help:Gallery_tag) which were done to the gallery tag which were introduced to MW since this result format was written will be a nice to have, too.

**Comment by Extarys** [comment]
Yes I tried to make a gallery from multiple subpage having a value like [[Has cover image::something.jpg]] but the result looks like a 2000 website :\
